README.md

```diff
@@ -85,38 +81,55 @@ websitePipeline
 
 ## Handling results
 
-Now we're going to add a basic rendering of the results to the page with integrated click tracking.
-This will redirect the user through the Sajari token endpoint to the real page identified by the result, registering their "click" on the way through.
+Now we're going to add a basic rendering of the results to the page with integrated event tracking.
+This will persist the tracking event against the current query id in localStorage and send the event data to Search.io to track how users use search results and/or move through an ecommerce purchase funnel.
 
 ```javascript
-const values = { q: "FAQ" };
-websitePipeline
-  .search(values, clickTrackedSession.next(values))
+const values = { q: "puppies" };
+pipeline
+  .search(values, { type: "EVENT", field: "id" })
   .then(([response, values]) => {
+    searchio.updateQueryId(response.queryId);
     response.results.forEach((r) => {
-      const title = document.createElement("a");
-      title.textContent = r.values.title;
-      title.href = r.values.url;
-      title.onmousedown = () => {
-        title.href = r.token.click;
+      const item = document.createElement("a");
+      item.textContent = r.values.name;
+      item.href = r.values.url;
+      item.onclick = () => {
+        searchio.track("click", r.id);
       };
 
-      document.body.appendChild(title);
+      document.body.appendChild(item);
     });
   })
   .catch((error) => {
     // Handle error...
   });
 ```
 
-## Consuming an interaction token
+## Tracking additional events
+
+When a user moves further through your purchase funnel (e.g. adding an item to their shopping cart, completing a purchase, etc) you'll want to track that also, as it will provide feedback to the ranking system. The correct query id will be preserved throughout the ecommerce funnel as long as an event with _type_ `click` was already tracked earlier.
+
+```javascript
+document.querySelector(".add-to-cart").addEventListener("click", (e) => {
+  searchio.track("add_to_cart", e.target.id);
+});
+
+document.querySelector(".complete-purchase").addEventListener("click", (e) => {
+  document.querySelectorAll(".cart-item").forEach((item) => {
+    searchio.track("purchase", item.id);
+  });
+});
+```
+
+## DEPRECATED - Consuming an interaction token
```

Comment:
Should we just remove this section??

Reply:
There's probably _some_ customers still using it, and if it's not documented here there isn't anywhere else it would be. I'm open to removing it though.

Reply:
Hmm yeah good point, maybe it is worth then just moving it to a different document that is linked from this readme. 

Just so that if you are looking for it you can find it, but if you are coming to this for the first time we aren't showing you docs for deprecated methods/solutions.

Reply:
That's definitely fair. I'll see if this docs builder we're using supports that.

Reply:
I can't see a way to easily do that, so I'll actually move this into the documentation on the `interactionConsume` method itself.